.gitignore

```diff
@@ -148,10 +148,4 @@ sdk/cosmosdb/cosmos/lib
 swagger/*.json
 
 # library-specific ignores
-sdk/keyvault/*/src/**/*.js
-
-# skipping browser tests while also ignoring the browser folders where the browser bundles are
-/**/browser/*.js
-/**/browser/*.js.map
-/**/browser/*.html
-!/test/browser/*.spec.ts
\ No newline at end of file
+sdk/keyvault/*/src/**/*.js
```

Comment:
why are js files getting put into the keyvault tree?

Also, it seems like if we need a rule like this, it should be in a .gitignore that is closer to the source of the problem?

Reply:
That is most certainly a separate issue. I'll make an issue to see how to clean up the .gitignore.

Reply:
The issue I'm going to make will be at the backlog for anyone to pick it up 😁

Reply:
Here's the issue for KeyVault: https://github.com/Azure/azure-sdk-for-js/issues/7370 I'll deal with it this month.